source/speech.py

```diff
@@ -436,8 +441,16 @@ def getIndentationSpeech(indentation):
 			res.append(symbol)
 		else:
 			res.append(u"{count} {symbol}".format(count=count, symbol=symbol))
-
-	return " ".join(res)
+		quarterTones += (count*4 if raw[0]== "\t" else count)
+
+		speak = indentConfig & config.INDENT_SPEECH
+		if indentConfig & config.INDENT_TONES: #Remove me during speech refactor.
+			if quarterTones <= MAX_SPACES:
+				pitch = BASE_FREQUENCY*2**(quarterTones/24.0)
+				tones.beep(pitch, TONE_DURATION)
+			else: #we have more than 72 spaces (24 tabs), and must speak it since we don't want to hurt the users ears.

```

Comment:
nit: Great comment, but please move it inside the block. We sometimes put tiny comments on the else line, but I think a comment like this makes more sense inside the block.